wikifier_service.py

```diff
@@ -6,13 +6,124 @@
 from flask import request
 from flask import send_from_directory
 from wikifier.wikifier import Wikifier
+import requests
+import time
 
 app = Flask(__name__)
 
 wikifier = Wikifier()
 config = json.load(open('wikifier/config.json'))
 
 
+
+
+@app.route('/rec', methods=['POST','GET'])
+def rec():
+   
+    #print(request.headers)
+    #print(request.data)
+    #print(request.form)
+    #print(request.args)
+    #print(request.url)
+    #print(request.__dict__)
+    #print(request.referrer)
+    #if flask.request.method == 'POST':
+    #    print(flask.request)
+
+
+    #deal with callback requests for general info
+
+    query = request.form.get('queries')
+
+    if query == None:
+         query = request.args.get('queries')
+    
+    if query == None:
+        callback = request.args.get('callback', False)
+        if callback:
+            content = str(callback) + '(' + str({
+           "name" : "Table-Linker Reconciliation for OpenRefine (en)",
```

Comment:
why do you need wikidata urls?

Reply:
the wikidata urls are for openrefine to put link on the candidates